hera_mc/mc_session.py

```diff
@@ -107,6 +107,36 @@ def _time_filter(self, table_object, time_column, starttime, stoptime=None,
 
         return result_list
 
+    def _special_insert(self, table_object, obj_list):
```

Comment:
This name is vague. Something like `_insert_ignoring_duplicates` would be better, I think.

Also, I think `table_object` is somewhat misnamed — if I'm reading correctly, this argument is a type. Something like `row_class`?

Reply:
`table_object` is referring to the Class that defines the table (inherits from DeclarativeBase). I'm happy to change the name, but I find the term `row` confusing. Maybe `table_class`?

Reply:
`table_object` is also what I use for the same thing in the `_time_filter` function, so I think I should change it there as well.